target/scripts/build/packages.sh

```diff
@@ -37,6 +37,9 @@ function _install_postfix
   chmod +x /bin/hostname
   apt-get "${QUIET}" install --no-install-recommends postfix
   mv /bin/hostname.bak /bin/hostname
+
+  # Instead of syslog, master.cf and main.cf we supply are configured to log to /dev/stdout
+  rm /etc/rsyslog.d/postfix.conf
```

Comment:
We could do that already at build time?

Reply:
This is at build time, right after the `postfix` package has been installed and created it :sweat_smile:

Reply:
Or did you mean changing `master.cf` and `main.cf`? Those could be co-located here, but later in the `Dockerfile` we'd overwrite them with our own copies, not the packages.

This `rm` line could alternatively be moved to start-up scripts if you want co-location.

Reply:
Oups, my fault. I just saw the script (`packages.sh`) instead of Dockerfile and forgot, that `packages.sh` is executed while building..